src/components/accounts/Register.tsx

```diff
@@ -30,6 +31,12 @@ const Register = () => {
   const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const { name, value } = e.target;
     setRegisterForm((val) => ({ ...val, [name]: value }));
+
+    if (e.target.value !== "") {
+      labelEle?.classList.add("active-label");
+    } else {
+      labelEle?.classList.remove("active-label");
```

Comment:
might be better to add `useState`? 
though for some reason  its not working with password